helm-frontend/src/components/Predictions.tsx

```diff
@@ -25,6 +25,69 @@ export default function Predictions({ predictions, requests }: Props) {
     return null;
   }
 
+  if (predictions && predictions[0] && predictions[0].base64_images) {
+    return (
+      <div>
+        <div className="flex flex-wrap justify-start items-start">
+          {predictions.map((prediction, idx) => (
+            <div className="w-full" key={idx}>
+              {predictions.length > 1 ? <h2>Trial {idx}</h2> : null}
+              <div className="mt-2 w-full">
+                <h3>
+                  <span className="mr-4">Prediction image</span>
+                </h3>
+                <div>
+                  {prediction &&
+                  prediction.base64_images &&
+                  prediction.base64_images[0] ? (
+                    <img
+                      src={"data:image;base64," + prediction.base64_images[0]}
+                      alt="Base64 Image"
+                    />
+                  ) : null}
+                </div>
+              </div>
+              <div className="accordion-wrapper">
```

Comment:
Everything to do with the prompt should be unified with the text predictions case - only the prediction text / image part has to change.